frontend/src/pages/routing/UnauthenticatedApp.tsx

```diff
@@ -2,12 +2,15 @@ import React from 'react';
 import { Route, Routes } from 'react-router-dom';
 import { CreateQuestion } from '../questions/CreateQuestion';
 import Questions from '../questions/Questions';
+import { UpdateQuestion } from '../questions/UpdateQuestion';
 
 const UnauthenticatedApp: React.FC = () => {
   return (
     <Routes>
       <Route path="questions/new" element={<CreateQuestion />} />
       <Route path="/" element={<Questions />} />
+      
+      <Route path="question/edit/:questionId" element={<UpdateQuestion />} />
```

Comment:
```suggestion
      <Route path="questions/:questionId/edit" element={<UpdateQuestion />} />
```

[7 restful routes](https://gist.github.com/alexpchin/09939db6f81d654af06b)